staging/src/kubevirt.io/api/core/v1/BUILD.bazel

```diff
@@ -18,6 +18,7 @@ go_library(
     importpath = "kubevirt.io/api/core/v1",
     visibility = ["//visibility:public"],
     deps = [
+        "//pkg/pointer:go_default_library",
```

Comment:
I don't think we can import `//pkg/..` packages in `/staging`.
It won't work.
Does `make && make generate` works in your local env?